docker/docker-compose.provider.yml

```diff
@@ -31,24 +31,21 @@ services:
           memory: 512M
     networks:
       - internal
-  caddy:
-    image: caddy:2
-    restart: unless-stopped
+  substrate:
+    image: prosopo/substrate-contracts-node:v0.29
     ports:
-      - "80:80"
-      - "443:443"
-      - "443:443/udp"
-    volumes:
-      - ./docker/provider.Caddyfile:/etc/caddy/Caddyfile
-      - caddy_data:/data
-      - caddy_config:/config
-    networks:
-      - external
+      - "9615:9615"
```

Comment:
Please separate this into a different docker compose file as the old one is valid for VPS setup.

Reply:
done